.github/workflows/ci.yml

```diff
@@ -42,15 +49,9 @@ jobs:
 
       - name: Test with pytest
         run: |
-          pip install pytest
+          pip install pytest "numpy${{ matrix.numpy-version }}"
           pytest --pyargs bottleneck
 
-  check:
-    needs: test
-    runs-on: ubuntu-latest
-    steps:
-      - name: Placeholder for CI checks in PRs
-        run: echo "Done"
```

Comment:
I think this is needed to avoid dangling checks in the PRs.

Reply:
How? It's just another superflous job next to the real `test` jobs.

Reply:
I think that's needed. Otherwise when we remove other tests (e.g. deprecating python versions), they will be dangling in the "missing" checks of the PR (as you can see right now for the `check` itself). 

I added the `check` to avoid changing the required checks everytime. I made the PR depends on it, such that whatever upstream step of the pipeline change, the PR will keep dependending on check and works as expected.

Reply:
I still think that there would be better ways to achieve what you are trying to accomplish with that dummy job, but whatever. This PR is not the place to discuss this. I reverted the removal.

Reply:
Happy to receive a PR to fix that. I'm not a Github expert, so I might surely missing something :)